command/version.go

```diff
@@ -1,21 +1,169 @@
 package command
 
 import (
+	"context"
 	"fmt"
 
 	"github.com/urfave/cli/v2"
 
+	"github.com/peak/s5cmd/log"
+	"github.com/peak/s5cmd/storage"
+	"github.com/peak/s5cmd/storage/url"
+	"github.com/peak/s5cmd/strutil"
 	"github.com/peak/s5cmd/version"
 )
 
+var versionHelpTemplate = `Name:
+	{{.HelpName}} - {{.Usage}}
+
+Usage:
+	{{.HelpName}} [options] s3://bucketname
+
+Options:
+	{{range .VisibleFlags}}{{.}}
+	{{end}}
+Examples:
+	1. Check the current version of the s5cmd
+	 	 > s5cmd {{.HelpName}}
+	
+	2. Get the bucket versioning status of a bucket
```

Comment:
I think these should not be part of `version` command.  The common practice of `version` subcommand is to display the version of the tool, these added features are related to `S3 object versions`.

Reply:
Added `bucket-version` command to configure bucket versioning. Removed bucket versioning related logic from the `version` command.